app/elements/candidates/invite-candidate-input.html

```diff
@@ -76,7 +67,30 @@
 			_removeItem: function(){
 				this.candidate.Removed = true;
 				this.remove();
-			}
+			},
+            _afterProperties: function(challenges, candidate){
+                var option = document.createElement("option");
+                if (this.challenges.length == 1){
```

Comment:
No, because i'd rather it be consistent. Select in both cases, just that in one there is only one already selected option(when you open this modal from the challenge-detail page).